packages/dashboard/package.json

```diff
@@ -55,7 +55,6 @@
     "react-dom": ">=16.3.2-0 <17.0.0"
   },
   "disabledTasks": [
-    "build-codepen-examples",
```

Comment:
Almost everything from here on out is the same couple changes repeated a bazillion times--I'll add comments calling out anything more interesting